### PR TITLE
[RAD-1989] Most basic test for StreetEdgeExporter

### PR DESCRIPTION
**Pull request by danielhfrank**
This adds a basic end-to-end test for `StreetEdgeExporter` - all that it really accomplishes is to make sure that we can run the exporter end to end, and that the output is valid and the expected size. I plan to use this to help me refactor the class into smaller chunks that we can more easily unit test (being able to ensure that it runs end to end after refactoring will provide some peace of mind).

I modeled this after the one other test that @marcopalmeri added. I moved some of the logic out of the `setUp` method and into the one test method because it was easier for me to trace logic that way. When I add other tests, I may choose to rethink that, and I'm also receptive to changing it now.

### CODE REVIEW

File: web/src/test/java/com/replica/StreetEdgeExporterTest.java [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+@ExtendWith(DropwizardExtensionsSupport.class)
+public class StreetEdgeExporterTest {
+
+    private static final String TARGET_DIR = "./target/gtfs-app-gh/";
+    private static final String TRANSIT_DATA_DIR = "transit_data/";
+    public static final DropwizardAppExtension<GraphHopperServerConfiguration> app = new DropwizardAppExtension<>(GraphHopperApplication.class, createConfig());
+    private static GraphHopperServerConfiguration createConfig() {
+        GraphHopperServerConfiguration config = new GraphHopperServerConfiguration();
+        config.getGraphHopperConfiguration().
+                putObject("graph.flag_encoders", "car,foot").
```
Comment (danielhfrank) [inline web/src/test/java/com/replica/StreetEdgeExporterTest.java:47]: This line is the only thing that differs between this setup code and the equivalent in the other test. I specified "car,foot" here, as well as in the yaml config file. Totally not clear to me why we do it in both places but 🤷‍♂️

File: web/src/test/java/com/replica/StreetEdgeExporterTest.java [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+    }
+
+    @AfterAll
+    public static void cleanUp() {
+        Helper.removeDir(new File(TARGET_DIR));
+        Helper.removeDir(new File(TRANSIT_DATA_DIR));
+    }
+
+    @Test
+    public void testSomething() throws IOException {
```
Comment (marcopalmeri) [inline web/src/test/java/com/replica/StreetEdgeExporterTest.java:74]: looks good.  but maybe rename the test to testExport() or something more descriptive?

**Review by marcopalmeri — APPROVED** [review]
one comment.  Thanks!

File: web/src/test/java/com/replica/StreetEdgeExporterTest.java (hunk 2 above)
Comment (danielhfrank) [reply web/src/test/java/com/replica/StreetEdgeExporterTest.java:74]: Oh, whoops, you caught me, that was just what I put down when I was getting started. Thanks!

File: web/src/test/java/com/replica/StreetEdgeExporterTest.java (hunk 1 above)
Comment (mark-idleman) [reply web/src/test/java/com/replica/StreetEdgeExporterTest.java:47]: Ahh, I see - in the other test, we actually use `app`; in this test, we copied over the same structure, but we never actually use this `app` variable. When we actually call the CLI commands below we're just passing the name of the config yaml, so I think the yamls are actually all we need, and this code at the beginning of this test can be removed

Comment (danielhfrank) [reply web/src/test/java/com/replica/StreetEdgeExporterTest.java:47]: Ah, thanks for looking at this with fresh eyes. When I was first trying to get this off the ground I was having the hardest time getting things to run / setup correctly, and so I copied over as much as I could. Now that I have things working, it's clearer that I can remove this section. Will include with my next phase of this work, thanks!

**Pull request merged**